addons/web/static/tests/chrome/action_manager_tests.js

```diff
@@ -583,6 +590,61 @@ QUnit.module('ActionManager', {
         actionManager.destroy();
     });
 
+    QUnit.test('executing a window action with onchange warning do not hide it', async function (assert) {
+        assert.expect(2);
+
+        this.data.partner.fields.o2m.onChange = true;
```

Comment:
is this useful??

Reply:
yes, we don't have an onchange without that

Reply:
mmmh 13.0, of course. As of 14.0, you won't need it anymore